recipes/Text_to_Shell/Text_to_Shell.ipynb

```diff
@@ -91,31 +214,79 @@
     "Question:\n",
     "create a tar ball of all pdf files in current folder\n",
     "Answer:\n",
-    "find . -name *.pdf | xargs tar czvf /root/Desktop/evidence/pdf.tar\n",
+    "find . -name *.pdf | xargs tar czvf pdf.tar\n",
     "\n",
     "Question:\n",
-    "Sort all files/directories under current directory according to modification time and print only the recent 7 of them\n",
+    "Sort all files/directories in the current directory, but no subdirectories, according to modification time and print only the recent 7 of them\n",
```

Comment:
Looks good. 

I noticed that the other example queries have grammatical errors; would you mind fixing those?

Reply:
Thanks for the catches! Yes, I'll go through them all again.